.github/workflows/coffee.yml

```
@@ -19,22 +19,18 @@ jobs:
       - name: Setup Go environment
         uses: actions/setup-go@v5
 
-      - name: Run Test and Send Slack Message
-        run: go run main.go coffee-break
-        env:
-          SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
-          HACBS_CHANNEL_ID: ${{ secrets.HACBS_CHANNEL_ID }}
-
-      - name: Commit and push
+      - name: Run Script, Commit and push
         run: |
-          git config --global --add safe.directory /__w/qe-tools/qe-tools
-          git init /__w/qe-tools/qe-tools
           git config user.name "GitHub Action"
           git config user.email "action@github.com"
           git switch -c coffee-break
+          go run main.go coffee-break
```

since with this change we are using the container, building a binary wouldn't be necessary, the binary should be already present in the image
```suggestion
          ./qe-tools coffee-break
```

---

Yes. I tried this but actions were not able to run it. It says no such file or directory, Probably not detecting the ".". I will check again